servicemap/rest_client.py

```diff
@@ -0,0 +1,44 @@
+import requests
+
+
+class ServicemapApiClient:
+
+    CONNECTION_TIMEOUT = 20
+    DATA_SOURCE = "tprek"
```

Comment:
Is the "tprek" the only datasource here? I set the id to be "tprek:{ID}" so that it would be in same format as it is in the Kultus database, in LinkedEvents API and in rest of the system.